ocs_sample_library_preview/OMF/Subscription.py

```diff
@@ -0,0 +1,97 @@
+import json
```

Comment:
`Subscription` and `Topic` objects contain a lot more properties, are we just consciously leaving them out since we don't currently use these? If so I don't necessarily have a problem with that, just want to make sure we're on the same page.

Reply:
That was my intention. I figured if someone requested those endpoints we could include them in a separate work item.

Reply:
I think that's OK. We've done that in the past.